website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
+title: Introduction to FLINT
+author: Ruchi Pakhle
+author_url: https://github.com/Ruchip16
+author_title: Passionate programmer, coder & a web-developer 
```

Comment:
Can we use a better author title here. Maybe an affiliation with the moja global community rather than being ambiguous :smile:

Reply:
Sure! I will change that